ui/src/app/app.constants.ts

```diff
@@ -113,5 +113,6 @@ export const Dataflow = {
 export const ColLength = {
   StorageMaxLength: 9223372036854775807,
   StringMaxLength: 2621440,
-  ByteMaxLength: 10485760
+  ByteMaxLength: 10485760,
+  DataTypes: ['STRING','BYTES','VARCHAR']
```

Comment:
Is this the right place for `DataTypes`?

Reply:
Placed it here because only these data types have column length associated with it